Mention Spatial -> Node3D rename in Introduction docs (#7338)
godotengine/godot-docs: getting_started/introduction/key_concepts_overview.rst

``Camera2D``, and a ``CollisionShape2D``.

.. note:: The node names end with "2D" because this is a 2D scene. Their 3D

>           counterparts have names that end with "3D". Be aware that there
>           was a renaming of "Spatial" Nodes to "Node3D" starting with Godot
>           4.

Reviewer: ```suggestion
          counterparts have names that end with "3D". Be aware that "Spatial"
          Nodes are now called "Node3D" starting with Godot 4.
```
I think this just reads a little cleaner as it avoids passive voice
